modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -2688,6 +2689,8 @@ private void _updateTable(
 
 			FinderCacheUtil.clearDSLQueryCache(
 				dynamicObjectDefinitionTable.getTableName());
+
+			objectEntryPersistence.clearCache(SetUtil.fromArray(objectEntryId));
```

Comment:
@TaiPhamLR Can we move this to `updateObjectEntry` instead? 

The `addOrUpdateExtensionDynamicObjectDefinitionTableValues` calls the `_updateTable` and in this case the `objectEntryId` is not the primary key of the ObjectEntry table (btw we should rename this), but the primary key of some System Object table, like Use or AccountEntry..